### DISCUSSION TITLE
how to use T5 in dlt?

### DISCUSSION THREAD

**Discussion by hongyinjie** [xhluca/dl-translate#45] [Q&A] [answered]
is there code sample...?
thanks...

I want to test it!

**Comment by xhluca** [comment, marked as answer]
It's already available on huggingface: https://twitter.com/Thom_Wolf/status/1244646461914169344/photo/1

I'm not very familiar with T5's translation capabilities (i know you can use prompt but not sure the coverage and accuracy), and it also seems it mostly only works well with larger models, which is hard to test for this library. If you are interested in contributing by adding the model + tests to this project I'm happy to welcome a PR :)

**Reply by xhluca** [reply]
I don't think T5 is a good option for translation right now, there's no widely available model specifically finetuned for multilingual translation. If you happen to find one, feel free to reopen this issue and contribute a PR!
